src/containers/SettingList/Providers/Providers.test.tsx

```diff
@@ -47,10 +46,10 @@ describe('<Providers />', () => {
     expect(getByText('Loading...')).toBeInTheDocument();
     await waitFor(() => {
       const checkbox = screen.getByRole('checkbox');
-      UserEvent.click(checkbox);
+      fireEvent.click(checkbox);
```

Comment:
Let's discuss switching to fireEvent instead UserEvent. According to doc UserEvent is recommended. Please check https://testing-library.com/docs/user-event/intro/#difference-to-fireevent